### PR TITLE
bugfix for inputing date with calendar plugin

### LINKED ISSUE #241: Inserting date from calendar broken with python3 (on Arch linux at least)
I'm using vim-orgmode with vim 7.4 (patches 1-1797) on arch linux (which has `python` default to python3). When I try to insert a date from the Calendar I get several errors. The first is easy to fix (in `org.vim:155` there is a `print` statement without parentheses, regardless if `s:py_version` is `python` or `python3`. After adding these parentheses, it still fails because `Date` is undefined. I'm not sure how to fix this (otherwise I would submit a pull-request). 

Thanks for a cool plugin!
Omri

### PR DESCRIPTION
**Pull request by akstrfn**
fixes #241

### CODE REVIEW

**Comment by omrihar** [comment]
Wow, great! Thaks for the super-quick response :-)

:+1:

File: ftplugin/org.vim [hunk 1]
```diff
@@ -152,7 +153,6 @@ fun CalendarAction(day, month, year, week, dir)
 	let msg = printf("Inserting %s | Modify date", g:org_timestamp)
 	exe s:py_version . "modifier = get_user_input('" . msg . "')"
 	" change date according to user input
-	exe s:py_version . "print modifier"
```
Comment (Ron89) [inline ftplugin/org.vim]: On line `155`. Why don't you simply `exe s:py_version . "print(modifier)"`?

File: ftplugin/org.vim [hunk 2]
```diff
@@ -115,6 +115,7 @@ for p in vim.eval("&runtimepath").split(','):
 from orgmode._vim import ORGMODE, insert_at_cursor, get_user_input, date_to_str
 ORGMODE.start()
 
+from Date import Date
```
Comment (Ron89) [inline ftplugin/org.vim]: On line118. Where did you use the imported `Date`?

Comment (akstrfn) [inline ftplugin/org.vim]: I made a mistake and removed it before, I haven't realised that it was used in calendar plugin... I somehow missed that one for a long time.

File: ftplugin/org.vim (hunk 1 above)
Comment (akstrfn) [inline ftplugin/org.vim]: To get the same behaviour in py2 you would need ``from __future__ import print_function``, and ``print modifier`` seemed unnecessary so I removed it.

**Pull request merged**